build_tools/android/common/benchmark_description.py

```diff
@@ -0,0 +1,280 @@
+#!/usr/bin/env python3
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utilities for describing Android benchmarks.
+
+This file provides common and structured representation of Android devices,
+benchmark definitions, and benchmark result collections, so that they can be
+shared between different stages of the same benchmark pipeline.
+"""
+
+import json
+import re
+import subprocess
+
+from dataclasses import dataclass
+from typing import Any, Dict, Sequence
+
+__all__ = ["AndroidDeviceInfo", "BenchmarkInfo", "BenchmarkResults"]
```

Comment:
Do we need to set this manually? I think we could just prefix the helper functions with underscore and then they will not be exported

Reply:
I prefer this way as it's more explicit. ;)